docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+- [Using the configtx.yaml file](#the-configtx-yaml-file)
+- [The orderer system channel](#the-orderer-system-channel)
+- [Creating an application channel](#creating-an-application-channel)
+- [Joining peers to the channel](#join-peers-to-the-channel)
+- [Setting anchor peers](#set-anchor-peers)
+
+## Setting up the configtxgen tool
+
+Channels are created by creating a channel genesis block that contains the initial configuration of the new channel. While it is possible to build these blocks manually, it is easier to use the [configtxgen](../commands/configtxgen.html) tool to write genesis blocks and channel creation artifacts. The tool works by reading a `configtx.yaml` file that defines the configuration of your channel. Before we discuss the `configtx.yaml` file in the next section, we can get started by downloading and setting up the `configtxgen` tool.
+
+You can download the `configtxgen` binaries by following the steps to [install the samples, binaries and Docker images](install.html). `configtxgen`, along with other Fabric tools, will be downloaded to the `bin` folder of your local clone of the `fabric-samples` repository.
+
+For the purposes of this tutorial, we will want to operate from the `test-network` directory inside `fabric-samples`. Navigate to that directory using the following command:
+```
+cd fabric-samples/test-network
+```
+We will operate from the `test-network` directory for the remainder of the tutorial. Use the following command to add the configtxgen tool to your CLI path:
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+```
+
+In order to use `configtxgen`, you need to the set the `FABRIC_CFG_PATH` environment variable to your local copy of the `configtx.yaml` file. For this tutorial, we will reference the `configtx.yaml` used to setup the Fabric test network in the `configtx` folder:
+```
+export FABRIC_CFG_PATH=${PWD}/configtx
+```
+
+You can check that you can are able to use the tool by printing the `configtxgen` help text:
+```
+configtxgen --help
+```
+
+## The configtx.yaml file
+
+The `configtx.yaml` file specifies the **channel configuration** that is contained inside the channel genesis block. The information that is required to build the channel configuration is specified in a readable and editable form in the `configtx.yaml` file. The `configtxgen` tool uses the channel profiles to create the channel configuration and write it to the protobuf format that can be read by Fabric.
+
+You can find the `configtx.yaml` file that is used to deploy the test network in the `configtx` folder in the `test-network` directory. The file contains the following information that we will use to create our new channel:
+
+- **Organizations:** Which organizations are part of the blockchain consortium, and consortium organizations will be members of your channel. Each organization has a reference to the cryptographic material is used to build the [channel MSP](../membership/membership.html).
```

Comment:
This may be better if it is simplified: The organizations that can become members of your channel. Each organization has a reference to the cryptographic material that is used to build the [channel MSP](../membership/membership.html)